R/load.R

```diff
@@ -64,8 +60,7 @@ load <- function(project = NULL, quiet = FALSE) {
   # then unload the current project and reload the requested one
   switch <-
     !renv_metadata_embedded() &&
-    !is.na(Sys.getenv("RENV_PROJECT", unset = NA)) &&
-    !identical(project, renv_project())
+    !renv_project_is_active(project)
```

Comment:
```suggestion
    !renv_project_loaded(project)
```

We might also need to do this stuff if any project is loaded (not just the current one)?